### PR TITLE
FEI-4888.1: Switch to Cypress for e2e tests

### PR DESCRIPTION
**Pull request by jandrade**
## Summary

This is a rollout of the Cypress e2e tests. This is the first
step in the rollout, which is to switch the e2e tests to run on the
Cypress infrastructure instead of the Python-Selenium infra.

The next step will be to remove the `e2e-cypress-*` jobs that were
created while we tested this integration in parallel with the
Python-Selenium infra.

Issue: FEI-4888

## Test Plan

- [x] Run the `e2e-test` job on this branch and verify that it passes in
  Jenkins via the "Replay" button.

NOTE: I created two separate Replay jobs to verify that `first` and
`second` e2e tests pass.

1. first-smoke-test: https://jenkins.khanacademy.org/job/deploy/job/e2e-test/47341/console
2. second-smoke-test: https://jenkins.khanacademy.org/job/deploy/job/e2e-test/47343/console

- [x] Run the `firstinqueue-priming` job on this branch and verify that
  it passes in Jenkins via the "Replay" button.

https://jenkins.khanacademy.org/job/deploy/job/firstinqueue-priming/3019/console

### CODE REVIEW

File: jobs/e2e-test.groovy
Comment (jandrade) [inline jobs/e2e-test.groovy:1]: **note:** I've copied the contents of [e2e-test-cypress.groovy](https://github.com/Khan/jenkins-jobs/blob/master/jobs/e2e-test-cypress.groovy) to this file.

The plan is to delete e2e-test-cypress in the next PR.

File: jobs/firstinqueue-priming.groovy [hunk 1]
```diff
@@ -99,42 +95,17 @@ def _setupWebapp() {
 
 def prime() {
    _setupWebapp();
-   // We also need to sync mobile, so we can run the mobile integration test
-   // (if we are assigned to do so).
-   // Note that Mobile latest code is at develop branch
-   // TODO(benkraft): Only run this if we get it from the splits?
-   kaGit.safeSyncToOrigin("git@github.com:Khan/mobile", "develop");
```
Comment (jandrade) [inline jobs/firstinqueue-priming.groovy:106]: **note:** These mobile integration tests are not supported by our Cypress infra/tests. If we still need to keep that around, we probably need to switch those tests to a different testing approach (e.g. integration tests in Go?).

**Review by jeresig — APPROVED** [review]
Great! This seems reasonable to me, thanks for adding the test plan.

File: jobs/firstinqueue-priming.groovy (hunk 1 above)
Comment (csilvers) [reply jobs/firstinqueue-priming.groovy:106]: Cc @jaredly and @jeresig , who I think set this up originally.  What do we want to do with legacy_e2e/activity/exercises/end_to_end/android_perseus_smoketest.py?  There's no analogue in cypress-land.

Though it looks like the test is currently disabled.  So maybe it's ok just to get rid of it?

Comment (jeresig) [reply jobs/firstinqueue-priming.groovy:106]: Yeah, I'm fine just getting rid of it!

**Review by csilvers — APPROVED** [review]
Looks good to me!

**Review by somewhatabstract — APPROVED** [review]
LGTM

**Review by dbraley — APPROVED** [review]
Ok, LGTM

**Pull request merged**